plugins/woocommerce/client/admin/client/settings-payments/onboarding/types.ts

```diff
@@ -4,6 +4,14 @@
 import { type ReactNode } from 'react';
 import { type RecommendedPaymentMethod } from '@woocommerce/data';
 
+/**
+ * Internal dependencies
+ */
+import {
+	Country,
+	MccsDisplayTreeItem,
+} from './providers/woopayments/steps/business-verification/types'; // To-do: Maybe move to @woocommerce/data
```

Comment:
Agree with this TODO, IMO we can do it as a follow up issue rather than as part of this PR because it's already quite large. 😄

Reply:
Sounds good! I’ll take care of this in a follow-up PR.